### PR TITLE
update helmet usage

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -83,8 +83,9 @@ try {
 
 // Use helmet to set security headers
 // only enable HSTS on heroku; Ops handles it in stage & prod configs
-const hsts = AppConstants.NODE_ENV === "heroku" ? true : false;
-app.use(helmet({ hsts }));
+if (AppConstants.NODE_ENV === "heroku") {
+  app.use(helmet.hsts({ maxAge: 63072000 }));
```
Comment (pdehaan) [inline server.js:87]: Q: Is this 2 years in seconds?

File: server.js [hunk 2]
```diff
@@ -98,10 +99,6 @@ if (AppConstants.NODE_ENV === "heroku") {
   SCRIPT_SOURCES.push(PONTOON_DOMAIN);
   STYLE_SOURCES.push(PONTOON_DOMAIN);
   FRAME_ANCESTORS.push(PONTOON_DOMAIN);
-  app.use(helmet.frameguard({
-    action: "allow-from",
-    domain: PONTOON_DOMAIN,
-  }));
```
Comment (pdehaan) [inline server.js:104]: Don't we need this anymore?

**Review by pdehaan — APPROVED** [review]

File: server.js (hunk 1 above)
Comment (groovecoder) [reply server.js]: Yeah. Updated to be more clear.

File: server.js (hunk 2 above)
Comment (groovecoder) [reply server.js:104]: helmet dropped `frameguard`, and observatory.mozilla.org looks at CSP `framce-ancestors` for this check now.

File: server.js [hunk 3]
```diff
@@ -181,6 +185,7 @@ app.use(session({
     maxAge: SESSION_DURATION_HOURS * 60 * 60 * 1000, // 48 hours
     rolling: true,
     sameSite: "lax",
+    secure: true,
```
Comment (groovecoder) [inline server.js:188]: Need to make this conditional on `NODE_ENV !== "dev"` since dev is on `http://localhost:6060`

**Comment by groovecoder** [comment]
@pdehaan another quick r?

File: server.js [hunk 4]
```diff
@@ -175,12 +179,14 @@ app.locals.LOGOS_ORIGIN = AppConstants.LOGOS_ORIGIN;
 app.locals.UTM_SOURCE = new URL(AppConstants.SERVER_URL).hostname;
 
 const SESSION_DURATION_HOURS = AppConstants.SESSION_DURATION_HOURS || 48;
+const SESSION_COOKIE_SECURE = AppConstants.NODE_ENV === "dev" ? false : true;
```
Comment (pdehaan) [inline server.js:182]: This works, or we could do `AppConstants.NODE_ENV !== "dev"` since that's already a Boolean.

**Review by pdehaan — APPROVED** [review]

File: server.js (hunk 4 above)
Comment (groovecoder) [reply server.js]: I like it! Updated.

**Pull request merged**